src/delegation_backend/app_config.go

```diff
@@ -84,15 +92,15 @@ func LoadEnv(log logging.EventLogger) AppConfig {
 	if config.Aws != nil {
 		configCount++
 	}
-	if config.Database != nil {
+	if config.AwsKeyspaces != nil {
 		configCount++
 	}
 	if config.LocalFileSystem != nil {
 		configCount++
 	}
 
 	if configCount != 1 {
```

Comment:
`configCount > 1` would be more appropiate here and `configCount < 1` for the case when no config is provided. Just friendly error messages.